### PR TITLE
Calculate circle circumference

### PR DESCRIPTION
**Pull request by Pentapatch**

### CODE REVIEW

File: ShitCodeApp/Data/Class2.cs [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace ShitCodeApp.Data
+{
+    internal class Class2
+    {
+        public int radius;
```
Comment (sorrz) [inline ShitCodeApp/Data/Class2.cs:11]: When using a CTOR, you can make the field private and use a GET SET, to protect it.

File: ShitCodeApp/Data/Class2.cs [hunk 2]
```diff
@@ -0,0 +1,18 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace ShitCodeApp.Data
+{
+    internal class Class2
```
Comment (sorrz) [inline ShitCodeApp/Data/Class2.cs:9]: Please review the naming convention of your class.

File: ShitCodeApp/Data/App.cs [hunk 3]
```diff
@@ -70,6 +70,13 @@ internal void Run()
                 Console.WriteLine(c);
                 Console.WriteLine(Calculate(12, NumberType.TenPercentOff, 10));
             }
+
+            CCC(new Class2(14));
+        }
+
+        public void CCC(Class2 c)
```
Comment (sorrz) [inline ShitCodeApp/Data/App.cs:77]: Please review your naming convention for the CCC and the Class and Variable

File: ShitCodeApp/Data/App.cs [hunk 4]
```diff
@@ -70,6 +70,13 @@ internal void Run()
...
+        {
+            double c2 = 2 * Math.PI * c.radius;
```
Comment (sorrz) [inline ShitCodeApp/Data/App.cs:79]: Please review your variable name "c2" to something more descriptive.

**Review by sorrz — CHANGES_REQUESTED** [review]
Please see my comments for changes needed to be made.

**Review by Pentapatch — COMMENTED** [review]
Updated to fix your suggestions

**Pull request merged**